packages/cli/package.json

```diff
@@ -29,6 +29,17 @@
     "p-limit": "^4.0.0",
     "strip-indent": "^4.0.0"
   },
+  "exports": {
+    "source": "./src/cli.ts",
+    "link": "./lib/commands/link.js",
+    "build": "./lib/commands/build.js",
+    "sync": "./lib/commands/sync.js"
+  },
+  "files": [
+    "lib/*",
+    "src/*",
+    "!*.test.*"
+  ],
```

Comment:
please put it before scripts field

Reply:
But in other packages like `css-data` `css-vars` the field is after the `scripts` one

Reply:
Made the changes 👍